data/committee/sortition/sortition_test.go

```diff
@@ -62,3 +68,57 @@ func TestSortitionBasic(t *testing.T) {
 		t.Errorf("wanted %d selections but got %d, d=%d, maxd=%d", expected, hitcount, d, maxd)
 	}
 }
+
+func TestCompareSortitionImpls(tt *testing.T) {
+	partitiontest.PartitionTest(tt)
+
+	// if TOTAL_MONEY env var is set, parse uint64 from env
+	var envTotalMoney uint64
+	if tm := os.Getenv("TOTAL_MONEY"); tm != "" {
```

Comment:
The env var is for configuring the unit test locally? Is it worthwhile ranging over more than one value in CI?

Reply:
So a non-blocking suggestions: consider removing the env-var logic as you can just hard code it as necessary locally.